src/edge/edgeManager.ts

```diff
@@ -504,6 +504,7 @@ export class EdgeManager {
                 }
             case Constants.LANGUAGE_NODE:
                 try {
+                    outputChannel.appendLine("Node.js Module creation may take some time.");

```

Comment:
better to have an estimation of the time?

Reply:
Updated